config/development-opa.yaml

```diff
@@ -0,0 +1,133 @@
+log:
+  stdout: true
+  level: info
+
+persistence:
+  defaultStore: cass-default
+  visibilityStore: cass-visibility
+  numHistoryShards: 4
+  datastores:
+    cass-default:
+      cassandra:
+        hosts: "127.0.0.1"
+        keyspace: "temporal"
+    cass-visibility:
+      cassandra:
+        hosts: "127.0.0.1"
+        keyspace: "temporal_visibility"
+global:
+  authorization:
+    authorizer: opa
```

Comment:
Instead of adding a new development config and adding to the development docker-compose, can you move all this into the https://github.com/temporalio/samples-server repo? You can add a new directory in there with a new docker-compose, configs file, and sample policy files. (Although I think you can get away without a new config file if you add a few template options to `docker/config_template.yaml` and set env vars in the docker-compose)

Reply:
Sounds good, I've removed all of that configuration from this repo and will create a PR in samples-server as soon as I can.